agent/exec/container/controller.go

```diff
@@ -141,7 +141,73 @@ func (r *controller) Start(ctx context.Context) error {
 		return errors.Wrap(err, "starting container failed")
 	}
 
-	return nil
+	// no health check
+	if ctnr.Config == nil || ctnr.Config.Healthcheck == nil {
+		return nil
+	}
+
+	healthCMD := ctnr.Config.Healthcheck.Test
+
+	if len(healthCMD) == 0 {
```

Comment:
nit: This should probably be `healthCmd` since it's an abbreviation not an acronym